srcpkgs/ryzenadj/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'ryzenadj'
+pkgname=ryzenadj
+version=0.2
+revision=1
+archs="x86_64*"
+wrksrc="RyzenAdj-${version}"
```

Comment:
the pkgname can be RyzenAdj

Reply:
I noticed that the manual said to use the same capitalisation as the dist archive file uses, but on the other hand, the binary itself is named `ryzenadj`. In my opinion, it's more intuitive if the binary and package have the same name. What do you think?